app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= turbo_frame_tag comment do %>
+  <div id="<%= dom_id comment %>">
+    <div class="shadow-xl mb-4 p-4">
+      <p>
+        <strong>Content:</strong>
+        <%= comment.content %>
+      </p>
+      <div class="flex flex-row">
+        <% if comment.user_id == current_user.id %>
```

Comment:
@aliraza4638 Adding conditions like this is always risky. Ideally there should be a single method which should be used by Pundit for authorization ?